engine/src/main/java/org/terasology/engine/EngineStatus.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.engine;
+
+/**
+ * Engine Status provides the current status of the engine - what it is doing, at a higher granularity than just running. This can be used by external and internal observers
+ * to report on the state of the engine, such as splash screens/loading screens.
+ *
+ * @author Immortius
+ */
+public interface EngineStatus {
+
+    /**
+     * @return The description of the status
+     */
+    String getDefaultDescription();
```

Comment:
I guess we'd have to decide what's the scope of language support. I.e. one thing is to make sure that the user interacts with a software in his/her language. It's another thing to ensure that developers can interact with a software in their own language, i.e. for debugging purposes.

If the status changes are mostly intended for developing/debugging purposes perhaps we can stick to English-only unless we want to go multilanguage at a deeper level. If the statuses are also intended to be meaningful for the user then I'd be ok with it I guess.